ws-android/Dockerfile

```diff
@@ -5,65 +5,180 @@ FROM cloud9/workspace
 # https://github.com/lukin0110/docker-android-build
 MAINTAINER Jeremy Ellis <keyfreemusic@gmail.com>
 
+# -------------------------- Startup Stuff -----------------------------------------
+# move files from the files folder to the docker container
 ADD ./files/workspace /home/ubuntu/workspace
 
-RUN chmod -R g+w /home/ubuntu/workspace && \
+# Give owner and ubuntu group: read,write,execute permissions, others read only
+# Change owner to ubuntu
+RUN chmod -R 774 /home/ubuntu/workspace && \
```

Comment:
Just testing if the permissions were the Gradle and Ant issues. I will switch these back